### PR TITLE
Add PowerShell example with -f flag to check reference

### LINKED ISSUE #2330: Document workaround for PowerShell scripts
### Description
PowerShell scripts do not return the right return code, as described in https://github.com/sensu/sensu/issues/1919.

### Category

This is an issue with:

- [ ] errata (error in doc content)
- [ ] new doc (requires a completely new doc)
- [x] update (add missing or refresh existing content)
- [ ] improvement (improve docs content or format)

### Affected pages
https://docs.sensu.io/sensu-go/latest/reference/checks/ ?

### Expected behavior
We should document the use of the `-f` flag, which appears to fix this issue.

### Current behavior
A user that doesn't specify this flag will only have return codes of `0` or `1`.

### Additional context

* Browser(s) and version(s): 
* OS:

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Adds a check example with PowerShell in command to check reference.

**Question**: Does the PowerShell workaround work with YAML? I included a yaml example, but the original issue only included JSON so I thought I should check.

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/2330

### CODE REVIEW

File: content/sensu-go/5.19/reference/checks.md [hunk 1]
```diff
@@ -1005,6 +1005,50 @@ spec:
 
 {{< /language-toggle >}}
 
+### PowerShell script in check commands
+
+If you use a PowerShell script in your check command, make sure to include the `-f` flag in the command.
```
Comment (rgeniesse) [inline content/sensu-go/5.19/reference/checks.md:1010]: May want to off some explanation into why the `-f` is required. My understanding of it is that it allows the proper exit code to be passed into Sensu.

Comment (hillaryfraley) [reply content/sensu-go/5.19/reference/checks.md:1010]: Good point, thank you. I will add it.

Comment (hillaryfraley) [inline content/sensu-go/5.19/reference/checks.md:1010]:
```suggestion
If you use a PowerShell script in your check command, make sure to include the `-f` flag in the command.
The `-f` flag ensures that the proper exit code is passed into Sensu.
```

**Review by palourde — APPROVED** [review]
👍

**Pull request merged**